metadata_service/proxy/atlas_proxy.py

```diff
@@ -428,7 +428,7 @@ def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
         parsed_reports = app.config['RESOURCE_REPORT_CLIENT'](reports) \
             if app.config['RESOURCE_REPORT_CLIENT'] else reports
 
-        return parsed_reports
+        return sorted(parsed_reports)
```

Comment:
can you please add test-case for this change ?

Reply:
yes, will do that